extensions/apps/extensions/src/components/pages/install-extension/install-extension-page.tsx

```diff
@@ -161,51 +214,63 @@ export const InstallExtensionPage = ({ appId }: { appId: string }) => {
     if (reportedError?.code) {
       return 'error';
     }
+    if (shouldRegisterResources && 1 < 0 /* successfullySigned */) {
+      return 'complete';
+    }
+    if (shouldRegisterResources) {
+      return 'authorize-request';
+    }
     return 'in-progress';
-  }, [isInstalled, reportedError]);
+  }, [isInstalled, reportedError?.code, shouldRegisterResources]);
 
   const progressInfo = useMemo(() => {
     if (isInstalled) {
-      return `${t('Application {{displayName}} successfully installed.', {
-        displayName: getAppDisplayName(data),
-      })}`;
+      return `${t(
+        'Extension {{displayName}} has been installed. The page will refresh and load it.',
+        {
+          displayName: selectAppDisplayName(data),
+        },
+      )}`;
     }
     if (reportedError?.code) {
       return getReportedError(reportedError, installerStatusCodes.current.error, t);
     }
     return getReportedProgress(reportedStatus, installerStatusCodes.current.status, t);
   }, [data, isInstalled, reportedError, reportedStatus, t]);
 
-  const action = useMemo(() => {
+  const successLabel = useMemo(() => {
     if (isInstalled) {
-      return {
-        label: t('Open the app'),
+      return t('Finished');
+    }
+    if (shouldRegisterResources && 1 < 0 /*successfullySigned*/) {
```

Comment:
there is something that wasn't updated here otherwise this will always be false

Reply:
we don't have the new resource sign flow, will be updated when we'll have it